pycode/memilio-epidata/memilio/epidata/defaultDict.py

```diff
@@ -51,7 +51,8 @@
     'file_format': 'json_timeasstring',
     'no_raw': False,
     'rep_date': False,
-    'sanitize_data': 1
+    'sanitize_data': 1,
+    'to_dataset': False
```

Comment:
Why not set this to True? I think it could improve the performance of some download tools a lot if the downloaded data can be used directly. Maybe open a new Issue for this.